executor/join_test.go

```diff
@@ -782,12 +782,9 @@ func (s *testSuite) TestIndexLookupJoin(c *C) {
 	tk.MustExec("CREATE TABLE `s` (`a` int, `b` char (20))")
 	tk.MustExec("CREATE INDEX idx_s_a ON s(`a`)")
 	tk.MustExec("INSERT INTO s VALUES (-277544960, 'fpnndsjo') ,  (2, 'kfpnndsjof') ,  (2, 'vtdiockfpn'), (-277544960, 'fpnndsjo') ,  (2, 'kfpnndsjof') ,  (6, 'ckfp')")
-	for _, enableChk := range []bool{true, false} {
```

Comment:
We'd better not to do this change. It will disable chunk execution and use the old `Next()` method. For now `Next()` has not been removed, it's still meaningful to test `Next()`

Reply:
DONE